### PR TITLE
Add per-level option to disable final stage string addition. Apply on Minecraft.

### PR DESCRIPTION
**Pull request by bcjordan**
The string addition from https://github.com/code-dot-org/code-dot-org/pull/5859 is something we can't add to the Minecraft tutorial. This adds an option to not add that text to the share dialog header and applies it to Minecraft levels
# Before

image
# After

image

### CODE REVIEW

**Comment by bcjordan** [comment]
@philbogle please review for inclusion, we'll want this to be in before https://github.com/code-dot-org/code-dot-org/pull/5859 goes out

File: apps/src/feedback.js [hunk 1]
```diff
@@ -565,8 +565,12 @@ FeedbackUtils.prototype.getFeedbackMessage_ = function(options) {
           puzzleNumber: options.level.puzzle_number || 0
         };
         if (options.feedbackType === TestResults.FREE_PLAY && !options.level.disableSharing) {
-          message = finalLevel ? msg.finalStage(msgParams) : '';
-          message = message + ' ' + options.appStrings.reinfFeedbackMsg;
+          if (options.level.disableFinalStageMessage) {
+            message = options.appStrings.reinfFeedbackMsg;
+          } else {
+            message = finalLevel ? msg.finalStage(msgParams) : '';
+            message = message + ' ' + options.appStrings.reinfFeedbackMsg;
```
Comment (philbogle) [inline apps/src/feedback.js]: Some other places in this file you defend against null appStrings but not here, should we add that?

Do you want the leading space even when message is ''?

**Comment by philbogle** [comment]
A couple comments.  LGTM after those are addressed.

**Comment by philbogle** [comment]
Looks good!

**Comment by laurelfan** [comment]
LGTM.

(the extra leading space doesn't cause problems if the string is used only in html)

**Pull request merged**